staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -225,12 +225,13 @@ const (
 	createTagFactor       = 2.0
 	createTagSteps        = 9
 
-	// encryptedCheck* is configuration of poll for created volume to check
+	// encryptedCheck* is configuration of exponential backoff for created volume to check
 	// it has not been silently removed by AWS.
 	// On a random AWS account (shared among several developers) it took 4s on
-	// average.
-	encryptedCheckInterval = 1 * time.Second
-	encryptedCheckTimeout  = 30 * time.Second
+	// average, 8s max.
+	encryptedCheckInitialDelay = 1 * time.Second
```

Comment:
Can you please rename the variables? They're used for generic volume creation check now.